lib/service-data/service-data.js

```diff
@@ -30,14 +30,30 @@ const external = {}
 
 external.getRuntimeData = getRuntimeData
 
-external.loadServiceData = () => {
+external.loadServiceData = async () => {
   const sourceObjs = external.getServiceSources()
   const schemas = external.getServiceSchemas()
-  return external.getRuntimeData(sourceObjs, schemas)
-    .then(serviceData => {
-      external.setServiceInstances(serviceData)
-      return serviceData
-    })
+
+  // temporary kludge to ensure renamed models are handled
+  if (schemas['definition.namespace']) {
+    process.stdout.write('switching models to namespace')
+    const servicePath = sourceObjs
+      .filter(sourceObj => sourceObj.source === 'service')
+      .map(sourceObj => sourceObj.path)[0]
+    const childProcess = require('child_process')
```

Comment:
All requires at the top of a file? Would be good to consider this rule: https://eslint.org/docs/rules/no-mixed-requires

Reply:
Because it's going to be removed once everything is migrated - and keeping it with the relevant code to zap